src/__tests__/helpers.ts

```diff
@@ -11,8 +15,19 @@ type SerializerType =
   | 'yaml'
   | 'yaml_depends_on'
   | 'dot'
-  | 'structure';
+  | 'structure'
+  | 'mutate';
 
+const mutate: MutatorFn = async (entity: Step) => {
```

Comment:
I'd have expected either an identity mutator so we can assert nothing changes, or better: a mutator that changes every step - let's say it generates a key based on the command contents as a salt to the randomizer? Then we can show that mutation works with all pipeline objects under test

Reply:
Just adding a string or a dummy command should be good enough no? We just want to test that each command is independently mutated, right?

Reply:
Changed this to not mutate anything like you originally suggested

Reply:
if we don't mutate anything not executing the code at all and running it doesn't show as a difference in the snapshots. Let's grab `seedrandom`, use the command value as a seed and produce a seeded hash that we can assign as a key. That should be quite easy, stable and will show in the snaps.

Reply:
done